extensions/libarchive/MergeContent.h

```diff
@@ -334,7 +334,9 @@ class MergeContent : public processors::BinFiles {
   std::string readContent(std::string path);
 };
 
-REGISTER_RESOURCE(MergeContent);
+REGISTER_RESOURCE(MergeContent, "Merges a Group of FlowFiles together based on a user-defined strategy and packages them into a single FlowFile. "
+    "It is recommended that the Processor be configured with only a single incoming connection, as Group of FlowFiles will not be created from "
```

Comment:
Processor is configured?

Reply:
"Processor is configured" would be passive voice, so not the best choice if we are to nit.